PurpleExplorer/Helpers/ServiceBusHelper.cs

```diff
@@ -99,35 +101,31 @@ public async Task SendTopicMessage(string connectionString, string topicPath, st
             await topicClient.CloseAsync();
         }
 
-        public void DeleteMessage(string connectionString, string topicPath, string subscriptionPath,
+        public async void DeleteMessage(string connectionString, string topicPath, string subscriptionPath,
```

Comment:
Should we return `Task`?
```suggestion
        public async Task DeleteMessage(string connectionString, string topicPath, string subscriptionPath,
```

Reply:
Done. thanks.